json-files/Michael

```diff
@@ -0,0 +1,11 @@
+​{
+"name": "Michael Demone",
+"email": "michael.demone@mail.utoronto.ca",
+"github": "mademone",
+"techskills": 3,
+"periods": ["ancient", "medieval", "early modern", "modern", "contemporary"],
+"regions": ["Europe", "North America"],
+"styles": ["political", "diplomatic", "cultural", "social", "art", "intellectual", "economic"],
+"picture": "../images/your-image.jpg",
```

Comment:
change this to a URL: 
- either upload an image of yourself to the "dist/images" folder and change this value to reflect the precise name of your image
- or, alternatively, point to the full URL of an existing image on the internet.

Reply:
I think I have completed this.